packages/common-all/src/engine/EngineV3Base.ts

```diff
@@ -298,7 +298,7 @@ export abstract class EngineV3Base implements ReducedDEngine {
     });
 
     if (items.length === 0) {
-      return { data: [] };
+      return [];
```

Comment:
why does `queryNotes` not return a `RespV3` object like all the other engine methods?

Reply:
(not a blocker - but we should consolidate the return methods in a subsequent PR if this isn't intentional)

Reply:
I wanted `queryNotes` to behave the same as `findNotes` which currently also returns a list of results. We can change it all to RespV3 later if you feel its necessary.